services/retrieve_and_rank/v1.js

```diff
@@ -250,7 +493,9 @@ module.exports = function (RED) {
       return node.error(message, msg);
     } else {
       (mode == 'delete' && Object.keys(res).length == 0) ? msg.payload = "Ranker deleted" : msg.payload = res;
-      node.send(msg);
+      if (mode != 'index') {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '!==' and instead saw '!='.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2405715173/issues/source?bid=3225115&fileBranchId=3233651#l496)